doc/classes/AudioEffectChorus.xml

```diff
@@ -14,78 +14,90 @@
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the cutoff frequency of a specific voice.
```

Comment:
It would be easiest if you commented directly on the code, like so, if you press the little button with a `+` and a `-` you can even suggest changes

Reply:
Ah great, thank you